dashboard/src/components/forms/DropdownButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement, useState } from "react";
+import Link from "next/link";
+
+interface DropdownButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  onClick?: () => void;
```

Comment:
@goldyydev Since you are already extending `React.ButtonHTMLAttributes<HTMLButtonElement>` you dont need to explicitly declare this
```suggestion
```